### PR TITLE
add yaml 0.1.0-wip

### PR DESCRIPTION
**Pull request by chinaunicomyangfan**
#### What type of PR is this?

Add one of the following kinds:

* enhancement/feature
Update ruc initial yaml file.Including regionUserCount api 

#### What this PR does / why we need it:
add initial yaml

### CODE REVIEW

**Review by philipxujin — APPROVED** [review]
ok

File: code/API_definitions/region-user-count.yaml
Comment (DanXu-ChinaTelecom) [inline code/API_definitions/region-user-count.yaml:1]: Maybe "switch-on users" need to change to  "switch on/connected UEs(User Equipments)"

Comment (DanXu-ChinaTelecom) [inline code/API_definitions/region-user-count.yaml:1]: Does this API just complete  one-time query？Do customers have the requirements of periodic query?

**Review by DanXu-ChinaTelecom — COMMENTED** [review]
Suggestion of connected User Equipments insteady of switch on users

**Comment by gregory1g** [comment]
Check chapter "2.5 Reduce telco-specific terminology in API definitions" of https://github.com/camaraproject/Commonalities/blob/main/documentation/API-design-guidelines.md.
"UE" and "MSISDN" are example of telco-specific terminology to be avoided.

**Comment by DanXu-ChinaTelecom** [comment]
@gregory1g Thanks，I see the issue discussion, there are two choice"Region Device Count" and "Region Sim Count" ,
I Check chapter "2.5 Reduce telco-specific terminology in API definitions" , suggest to use 'Terminal' rather than"UE",
regarding this, we may have three choice"Region Device Count" , "Region Sim Count" and "Region Terminal Count" . Perhaps, we can wait the view from TSC.

File: code/API_definitions/region-user-count.yaml
Comment (chinaunicomyangfan) [reply code/API_definitions/region-user-count.yaml:1]: Most scenarios of this interface require periodic or on-demand calls, such as [emergency rescue scenarios ](https://github.com/camaraproject/RegionUserCount/issues/10) or [outdoor live streaming scenarios](https://github.com/camaraproject/RegionUserCount/issues/11), where users need to repeatedly call to grasp real-time information on the number of devices in this area

**Pull request merged**